src/redprl/pretty.sml

```diff
@@ -267,10 +267,10 @@ struct
 
      | O.DIM0 $ _ => char #"0"
      | O.DIM1 $ _ => char #"1"
-     | O.MK_TUBE $ [_ \ r1, _ \ r2, [u] \ mu]  => 
+     | O.MK_TUBE $ [_ \ r1, _ \ r2, tube]  =>
        Atomic.squares @@ hsep
          [seq [ppTerm r1, Atomic.equals, ppTerm r2],
-          nest 1 @@ hvsep [Atomic.braces @@ ppVar u, ppTerm mu]]
+          nest 1 @@ ppBinder tube]
```

Comment:
@jonsterling Is there a reason not to group them?

Reply:
I can't remember